src/extension/_locales/en/messages.json

```diff
@@ -173,6 +173,9 @@
   "config_invalid_host": {
     "message": "Invalid hostname"
   },
+  "config_invalid_mountpoints": {
+    "message": "Invalid Content URL"
```

Comment:
@rgeraghty can you add the translations for this new english item?

Reply:
@rofe hope to have translation tomorrow,